dynamic-plugins/_utils/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "dynamic-plugins-utils",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "backstage": {
+    "role": "cli"
+  },
+  "scripts": {
+    "copy-dynamic-plugins": "node ./copy-plugins.js",
```

Comment:
Why rename this?

```suggestion
    "copy-plugins": "node ./copy-plugins.js",
```

Reply:
because when called from the root, through yarn, it gives more context.